ui/package.json

```diff
@@ -24,6 +24,7 @@
     "@mdx-js/runtime": "^1.0.20",
     "@mozilla-frontend-infra/components": "^2.4.0",
     "@sentry/browser": "^5.0.0",
+    "@types/react": "16.8.0",
```

Comment:
Why is this needed?

Reply:
It was auto-generated, After I upgraded `Yarn`. 
Also after using this `yarn.lock` and `package.json `, Number of warnings have been reduced to **1**

Reply:
`yarn.lock` is autogenerated but I don't think `ui/package.json` is. You can try doing `yarn remove @types/react`.